community/values/src/main/java/org/neo4j/values/storable/CharValue.java

```diff
@@ -68,6 +69,18 @@ public int computeHash()
         return 31 + value;
     }
 
+    @Override
+    public long updateHash( HashFunction hashFunction, long hash )
+    {
+        return updateHash( hashFunction, hash, value );
+    }
+
+    public static long updateHash( HashFunction hashFunction, long hash, char value )
+    {
+        hash = hashFunction.update( hash, value );
+        return hashFunction.update( hash, 1 ); // Pretend we're a string of length 1.
```

Comment:
As we are pretending to be a single character String here, do we need to hash the code-points of the char instead?

Reply:
`computeHash` does the same thing, so I assume it's fine.